BootloaderCommonPkg/Library/TpmLib/TpmLib.c

```diff
@@ -822,7 +822,7 @@ TpmExtendSecureBootPolicy (
 
   Data = FeaturePcdGet (PcdVerifiedBootEnabled);
 
-  Status = TpmHashAndExtendPcrEventLog (7, &Data, sizeof (Data), EV_EFI_VARIABLE_DRIVER_CONFIG, sizeof ("SecureBootPolicy"), (UINT8*)"SecureBootPolicy");
+  Status = TpmHashAndExtendPcrEventLog (7, &Data, sizeof (Data), EV_PLATFORM_CONFIG_FLAGS, sizeof ("SecureBootPolicy"), (UINT8*)"SecureBootPolicy");
```

Comment:
EV_PLATFORM_CONFIG_FLAGS will be used in PCR[1] only.
This is secure boot config enable/disable one.
We can check EV_EFI_VARIABLE_AUTHORITY.

Reply:
as discussed, we can can go with EV_PLATFORM_CONFIG_FLAGS and PCR[7].